### DISCUSSION TITLE
How to have the first suggestion not highlighted/selected initially?

### DISCUSSION THREAD

**Discussion by kujohn** [company-mode/company-mode#1158] [Q&A] [answered]
Is there a way to not have the suggestions selected initially so that pressing enter allows the carriage return? Only when hitting tab will use the selection.

**Comment by yugaego** [comment, marked as answer]
If I got you right, a [similar question](https://github.com/company-mode/company-mode/issues/913) has been resolved by using [`company-tng`](https://github.com/company-mode/company-mode/blob/master/company-tng.el).

To quick-test if it works for you run the following command: `M-x company-tng-mode`.

**Reply by kujohn** [reply]
`company-tng` works beautifuly, thanks!

**Comment by dgutov** [comment]
Or if `company-tng` is too much, you can try just changing `company-selection-default` to nil.

There is an example of such usage here: https://github.com/company-mode/company-mode/issues/460#issuecomment-837870120
